devtools/conda-envs/full-stack.yaml

```diff
@@ -24,15 +24,16 @@ dependencies:
 
     # Recharge
   - qcelemental
-  - qcportal ~=0.50
+  - qcportal ~=0.56
   - qcengine
   - xtb-python
   - geometric =1
 
-  - openff-qcsubmit ~=0.50
+  - openff-qcsubmit ~=0.52
```

Comment:
I don't know anything about QCPortal/QCSubmit compatibility in these version ranges, this is just the most recent release